apps/nextra/pages/en/build/smart-contracts/error-codes.mdx

```diff
@@ -23,6 +23,37 @@ Help us update this list by sending pull requests containing the errors you
 encounter. If you don't know how to resolve the error, as described int the
 _Action_ column, simply leave it blank.
 
+## Frequent Errors
+
+### INSUFFICIENT_BALANCE_FOR_TRANSACTION_FEE
+
+This means that the highest possible gas used for the transaction is greater than
+the balance of APT in the transaction sender's (or fee payer's) account.  To resolve,
+please submit with a lower max gas amount, and try again.
+
+For example, if the max gas amount is 1000, and the gas unit price is 100, the total
```

Comment:
Would be nice to say what are units of 1000, 100 (octas?) and how we get to decimals from these two numbers?